src/pages/material/news-details/news-details.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import { useStyles } from './news-details.styles';
+import NewsForm from '../../../components/news-form';
+import LoadingBar from '../../../components/loading-bar';
+
+import { getArticle } from '../../../redux/news/news.actions';
+
+const NewsDetails = ({ match }) => {
+  const { id } = match.params;
+
+  const dispatch = useDispatch();
+  const { loading, newsArticle } = useSelector(({ News }) => ({
+    loading: News.newsLoading,
+    newsArticle: News.newsArticle
+  }));
+  const classes = useStyles();
+
+  useEffect(() => {
+    dispatch(getArticle(id));
+  }, [dispatch, id]);
+
+  if (loading) {
+    return <LoadingBar />;
+  }
+
+  return (
+    <div className={classes.detailsContainer}>
+      {newsArticle !== null ? <NewsForm id={id} article={newsArticle} /> : null}
```

Comment:
It's easier to implement this check like this: `newsArticle ?` instead of `list !== null `